_posts/2023-11-20-testing-quarkus-with-citrus.adoc

```diff
@@ -0,0 +1,506 @@
+---
+layout: post
+title: 'Testing Quarkus with Citrus'
+date: 2023-11-20
+tags: testing event-driven integration kafka messaging
+synopsis: 'Explore how to verify Quarkus event-driven applications with the Citrus integration test framework.'
+author: christophd
```

Comment:
you need to create an entry for this author in  `_data/authors.yaml` otherwise it shows up author-less.

Reply:
ah good point, thanks. let me add this

Reply:
Done